### PR TITLE
Seed database with ApiUsers

### PR DESCRIPTION
**Pull request by sibsmc**
Added a seeds.rb file which populates database with ApiUsers.

### CODE REVIEW

**Comment by sibsmc** [comment]
Addresses issue #156

File: server/db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ApiUser.destroy_all
+
+ApiUser.create([
+                 {first_name: 'Mary', last_name: 'Black', city: 'Dublin', mentor: true, mentee: false, email: 'Test_email_1@email.com', password_digest: 'password1'},
```
Comment (diwanow) [inline server/db/seeds.rb]: I'd suggest to use @example.com (reserved domain that apparently bounces emails but at least safe; some more info [here](https://stackoverflow.com/questions/1368163/is-there-a-standard-domain-for-testing-throwaway-email/1368309) ); ideally it should be an owned domain, but not sure if we have one.

Reason: if we send emails, they should not arrive to someone's inbox. If email is used for security purposes, then someone with access to this external inbox can gain access to the system - quite bad if admin access. Therefore, the above is a general recommendation.

File: server/db/seeds.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+ApiUser.destroy_all
+
+ApiUser.create([
+                 {first_name: 'Mary', last_name: 'Black', city: 'Dublin', mentor: true, mentee: false, email: 'Test_email_1@email.com', password_digest: 'password1'}, 
+                 {first_name: 'Bob', last_name: 'Dylan', city: 'New York', mentor: true, mentee: false, email: 'Test_email_2@email.com', password_digest: 'password2'}, 
+                 {first_name: 'Jane', last_name: 'Smith', city: 'Edinburgh', mentor: false, mentee: true, email: 'Test_email3@email.com', password_digest: 'password3'}
```
Comment (diwanow) [inline server/db/seeds.rb]: Minor: I think we had someone to be both a mentor and a mentee as a valid scenario. How about one of these 3 users is both mentor and a mentee so all these options are covered?

File: server/db/seeds.rb (hunk 1 above)
Comment (sibsmc) [reply server/db/seeds.rb]: Cool didn't know about this, will use @example.com for now, we don't need to test anything with emails yet.

File: server/db/seeds.rb (hunk 2 above)
Comment (sibsmc) [reply server/db/seeds.rb]: Yes that's a good point.

**Pull request merged**